### PR TITLE
add additional info to flagNoInit

### PR DESCRIPTION
**Pull request by akmal-spacelift**
internal ref: https://spacelift-io.slack.com/archives/C06F1SB0LN5/p1726242982411049

### CODE REVIEW

**Comment by akmal-spacelift** [comment]
cc @KNiepok @eliecharra

File: internal/cmd/stack/flags.go [hunk 1]
```diff
@@ -62,7 +62,7 @@ var flagRunLatest = &cli.BoolFlag{
 
 var flagNoInit = &cli.BoolFlag{
 	Name:  "noinit",
-	Usage: "Indicate whether to skip initialization for a task",
+	Usage: "Indicate whether to skip initialization for a task.  Initializing the workspace before running a task means executing all the pre-initialization hooks as well as running the vendor-specific initialization procedure",
```
Comment (tomasmik) [inline internal/cmd/stack/flags.go:65]:
```suggestion
	Usage: "Indicate whether to skip initialization for a task. Initializing the workspace before running a task means executing all the pre-initialization hooks as well as running the vendor-specific initialization procedure",
```

You got two spaces there

Comment (tomasmik) [reply internal/cmd/stack/flags.go:65]: Could this actually just say `Any pre-initialization hooks as well the vendor-specific initialization procedure will be skipped"?

Comment (eliecharra) [reply internal/cmd/stack/flags.go:65]: @tomasmik's wording suggestion sounds more straightforward to me 👍🏻

**Review by eliecharra — APPROVED** [review]

**Pull request merged**